### ISSUE TITLE
Can't start training with custom data on Colab notebook

### ISSUE THREAD

**Issue by Alessio-Parmeggiani** [nerfstudio-project/nerfstudio#1746]
I used a copy of the notebook provided in the main webpage: [Colab link](https://colab.research.google.com/github/nerfstudio-project/nerfstudio/blob/main/colab/demo.ipynb)
The colab works perfectly when using preset data (I tested Dozer) and works also when uploading custom images and performing the preprocessing using COLMAP.
The issue appears when using custom images with a custom transform.json file.

In this case the training starts in the notebook, it outputs the config files and then finish without starting the training.
To start the training I used:     
`!ns-train nerfacto --viewer.websocket-port 7007 nerfstudio-data --data data/nerfstudio/custom_data --downscale-factor 4`

There are no error in the output, the last line that is sent in output is: 
`[06:52:48] Saving config to: outputs/unnamed/nerfacto/2023-04-17_065247/config.yml   `

I uploaded my images in the same format provided by the notebook:

```
-data/nerfstudio/custom_data
   -transforms.json
   -images 
        -rgb
            -1.jpg
            -2.jpg
            -...
```

And the transforms.json is in the correct form (copied from the dozer demo):
```
{
  "fl_x": 1066.0418701171875,
  "fl_y": 1066.0418701171875,
  "cx": 938.5834350585938,
  "cy": 604.4529418945312,
  "w": 1920,
  "h": 1080,
  "k1": 0.0,
  "k2": 0.0,
  "k3": 0.0,
  "k4": 0.0,
  "p1": 0.0,
  "p2": 0.0,
  "frames": [
    {
      "file_path": "./images/rgb/1.jpg",
      "transform_matrix": [...]
   }
....
```
Do you know how can I solve the problem or why the training is not starting?

I already tried the solution proposed in https://github.com/nerfstudio-project/nerfstudio/issues/1432, that is:
use `!pip install nerfstudio`
instead of `!pip install git+https://github.com/nerfstudio-project/nerfstudio.git`

**Comment by Alessio-Parmeggiani** [comment]
I already tried the solution proposed in  https://github.com/nerfstudio-project/nerfstudio/issues/1432, that is:
use `!pip install nerfstudio`
instead of `!pip install git+https://github.com/nerfstudio-project/nerfstudio.git`

**Comment by Alessio-Parmeggiani** [comment]
I already tried the solution proposed in https://github.com/nerfstudio-project/nerfstudio/issues/1432, that is:
use !pip install nerfstudio
instead of !pip install git+https://github.com/nerfstudio-project/nerfstudio.git

**Comment by Alessio-Parmeggiani** [comment]
I already tried the solution proposed in https://github.com/nerfstudio-project/nerfstudio/issues/1432, that is:
use !pip install nerfstudio
instead of !pip install git+https://github.com/nerfstudio-project/nerfstudio.git
But it didn't work

**Comment by Alessio-Parmeggiani** [comment]
> Hi, does the cell not create any output or does the cell end and return?
> 
> There is currently a bug in Google Colab where the output of training is very delayed or might not output at all. However, if the cell is running it is possible that the viewer just was not working, which was fixed in #1759.

Hi, the cell ends without errors after printing and saving the model config and the viewer says renderer disconnected.
![output](https://user-images.githubusercontent.com/57560488/232782613-fb40f8b3-a89d-4840-9f97-2708efd86113.jpg)

**Comment by Alessio-Parmeggiani** [comment]
I found the solution!
The problem was given by the command used:
  `!ns-train nerfacto --viewer.websocket-port 7007 nerfstudio-data --data data/nerfstudio/$scene --downscale-factor 4`
  
Here `--downscale-factor 4` means  that there should be a folder called "image_4" with the images downscaled (I think) that I did not have. Removing that argument solved the problem.
(It took so long to find the error because you had also to scroll the cell sideways to see the downscale factor).

It's strange that there are no error messages if the images folder does not exist
